frontend/components/Domain/QueryFilterBuilder.vue

```diff
@@ -0,0 +1,630 @@
+          relOps.value[part.relationalOperator] : field.relationalOperatorValue;
+
+        if (field.leftParenthesis || field.rightParenthesis) {
+          state.showAdvanced = true;
+        }
+
+        if (field.fieldOptions?.length || isOrganizerType(field.type)) {
+          if (typeof part.value === "string") {
+            field.values = part.value ? [part.value] : [];
+          } else {
+            field.values = part.value || [];
+          }
+
+          if (isOrganizerType(field.type)) {
+            hydrateOrganizers(field, index);
+          }
+
+        } else if (field.type === "boolean") {
+          const boolString = part.value || "false";
+          field.value = (
+            boolString[0].toLowerCase() === "t" ||
+            boolString[0].toLowerCase() === "y" ||
+            boolString[0] === "1"
+          );
+        } else if (field.type === "number") {
+          field.value = Number(part.value as string || "0");
+          if (isNaN(field.value)) {
+            error = true;
+            return initFieldsError(`Invalid query filter; invalid number value "${(part.value || "").toString()}"`);
+          }
+        } else if (field.type === "date") {
+          field.value = part.value as string || "";
+          const date = new Date(field.value);
+          if (isNaN(date.getTime())) {
+            error = true;
+            return initFieldsError(`Invalid query filter; invalid date value "${(part.value || "").toString()}"`);
+          }
+        } else {
+          field.value = part.value as string || "";
+        }
+
+        initFields.push(field);
+      });
+
+      if (initFields.length && !error) {
+        fields.value = initFields;
+      } else {
+        initFieldsError();
+      }
+    };
+
+    try {
+      initializeFields();
+    } catch (error) {
+      initFieldsError(`Error initializing fields: ${(error || "").toString()}`);
+    }
+
+
+    const attrs = computed(() => {
+      const baseColMaxWidth = 55;
+      const attrs = {
+        col: {
+          class: "d-flex justify-center align-end field-col pa-1",
+        },
+        select: {
+          textClass: "d-flex justify-center text-center",
+        },
+        fields: {
+          icon: {
+            cols: 1,
+            style: "width: fit-content;",
+          },
+          leftParens: {
+            cols: state.showAdvanced ? 1 : 0,
+            style: `min-width: ${state.showAdvanced ? baseColMaxWidth : 0}px;`,
+          },
+          logicalOperator: {
+            cols: 1,
+            style: `min-width: ${baseColMaxWidth}px;`,
+          },
+          fieldName: {
+            cols: state.showAdvanced ? 2 : 3,
+            style: `min-width: ${state.showAdvanced ? baseColMaxWidth * 2 : baseColMaxWidth * 3}px;`,
+          },
+          relationalOperator: {
+            cols: 2,
+            style: `min-width: ${baseColMaxWidth * 2}px;`,
+          },
+          fieldValue: {
+            cols: state.showAdvanced ? 2 : 3,
```

Comment:
I feel like we should use all availlable space, i would suggest on increasing the size for fieldName or maybe the relationalOperator by 1 (depends on if there are long translations and whether we want to increase the width of the edit window)
```suggestion
            cols: state.showAdvanced ? 3 : 4,
```

Reply:
Grids can only be exactly 12 columns, so we can't increase the space on one field without removing it from another. I think the other suggestions should help a lot (working on those!)

Reply:
This is currently using 11 columns

Reply:
LOL. Thanks